src/test/scala/uk/gov/hmrc/play/views/layouts/TrackingConsentSnippetSpec.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.play.views.layouts
+
+import org.scalatest.{Matchers, WordSpec}
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.Application
+import uk.gov.hmrc.play.JsoupHelpers
+import uk.gov.hmrc.play.views.html.layouts.trackingConsentSnippet
+
+class TrackingConsentSnippetSpec extends WordSpec with Matchers with GuiceOneAppPerSuite with JsoupHelpers {
+  override def fakeApplication(): Application =
+    new GuiceApplicationBuilder()
+      .configure(Map(
+        "play.allowGlobalApplication"             -> "true",
```

Comment:
So, am I right in thinking that this config key is to test the Play 2.5 non-DI versions of the template, where we need to the current running application? If so, is it worth doing what we have done in other projects and splitting these tests out into the existing Play 2.5 and Play 2.6 folders? In Play 2.5 include:
```
play.allowGlobalApplication"             -> "true"
```

In Play 2.6, explicitly flag it off:
```
play.allowGlobalApplication"             -> "false"
```

Reply:
As discussed, we will revisit this test suite when we deprecate and remove Play 2.5 support.

Reply:
As discussed, this wouldn't add value so consider this comment closed.